Practica1/firstDeliverable.go

```diff
@@ -0,0 +1,169 @@
+package main
+
+import (
+	"encoding/csv"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+)
+
+type data struct {
+	ID   int
+	Name string
+}
+
+func main() {
+	http.HandleFunc("/readCSVFile", handlerReadFile)
```

Comment:
I suggest to use a framework to define your router (gin, echo, chi, gorilla/mux, etc)